src/ts/ConvivaAnalytics.ts

```diff
@@ -160,15 +204,24 @@ export class ConvivaAnalytics {
     this.convivaAnalyticsTracker.resumeTracking();
   }
 
-  public release(): void {
-    this.destroy();
-    this.convivaAnalyticsTracker.release();
-  }
+  public release(event?: PlayerEventBase): void {
```

Comment:
I think this change is 'API breaking'? 🤔

Reply:
Yeah, I noticed that as well while you were reviewing, here is a fix https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/115/commits/f25bc277b73fade37d6832969805079ae99f089e. 🙂